### DISCUSSION TITLE
[Bug] Creating issue with a template from yaml file doesn't show on GitHub mobile.

### DISCUSSION THREAD

**Discussion by rafidalhaque** [community/community#8748] [Mobile] [answered]
Hello,
I create a issue template by configuring yaml file in a repository. But when I want to create issue (for test) in that repo the template from yaml file doesn't show on theme chooser. Please fix it.

**Repo:** [github.com/CrescentOS-repo/cos-salam](github.com/CrescentOS-repo/cos-salam)

**Comment by aconsuegra** [comment, marked as answer]
Hi @rafidalhaque,
thanks for your feedback. We included basic support for Issue Forms (those configured using a yaml file) in our latest beta versions. It will be possible to see those templates and they will redirect to our mobile website when they are selected. We will include full native support eventually.
